src/Hangfire.Mongo.Tests/MultipleServersFacts.cs

```diff
@@ -69,5 +70,53 @@ public void MultipleServerRunsRecurrentJobs()
             }
         }
 
+
+        [Fact, CleanDatabase]
+        public void MultipleBackgroundJobServers_AddsRecurrentJobs()
+        {
+            // ARRANGE
+            const int serverCount = 15;
+            const int workerCount = 2;
+
+            JobStorage.Current = ConnectionUtils.CreateStorage(new MongoStorageOptions());
+
+            var options = Enumerable.Range(0, serverCount)
+                .Select((_, i) => new BackgroundJobServerOptions
+                {
+                    Queues = new[] { "default", $"queue_{i}" },
+                    WorkerCount = workerCount
+                })
+                .ToList();
+
+            var servers = options.Select(o => new BackgroundJobServer(o)).ToList();
+
+            // let hangfire run for 1 sec
+            Task.Delay(1000).Wait();
+
+            // ACT
+            foreach (var queue in options.SelectMany(o => o.Queues))
+            {
+                for (int i = 0; i < workerCount; i++)
+                {
+                    RecurringJob.AddOrUpdate($@"job_{queue}.{i}-a", () => Console.WriteLine($@"{queue}.{i}-a"), Cron.Minutely(), null, queue);
```

Comment:
The static helper will choose which BackgroundServer this job goes to depending on the queue?